tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -24,7 +24,7 @@ impl NoiseLevel {
     // for formats which did not have noise levels saved
     pub const MAX: Self = Self(usize::MAX);
     // As a safety measure the unknwon noise level is set to the max value
-    pub const UNKOWN: Self = Self::MAX;
+    pub const UNKNOWN: Self = Self::MAX;
```

Comment:
there is also a type in the comment above

Reply:
arf thanks

Reply:
fixed thanks